src/timesync/timesyncd-server.c

```diff
@@ -92,6 +92,14 @@ int server_name_new(
                 break;
         case SERVER_LINK:
                 LIST_APPEND(names, m->link_servers, n);
+                // If we add a new link server, emit changed event
```

Comment:
as per coding style we reserve // style comments for "FIXME" style comments, and use /**/ for all explanatory comments.